dask/dataframe/core.py

```diff
@@ -253,6 +253,10 @@ def npartitions(self):
         return len(self.divisions) - 1
 
     @property
+    def size(self):
+        return self.reduction(lambda x: x.size, np.sum, token='size', meta=int)
```

Comment:
In an effort to keep lambda's out of non-user defined graphs, can you move this functionality to the `methods.py` file, as done here: https://github.com/dask/dask/blob/master/dask/dataframe/methods.py#L73-L74?